libraries/lib-registries/Registry.h

```diff
@@ -300,7 +355,16 @@ namespace detail {
       Literal mRoot;
    };
 
-extern template struct GroupItem<>;
+#ifndef _WIN32
+// extern explicit instantiation to avoid link time de-duplication
+extern
```

Comment:
There is [evidence](https://www.zverovich.net/2023/04/10/cxx20-modules-in-clang.html) that extern templates do not really work on Clang, the code is quite spooky on CL. Do we need them here? What is the measured effect of this?

Reply:
I needed special code for Windows to link.  I might make it compile nothing otherwise.